docs/latest/anatomy-of-a-tool/strategies.md

```diff
@@ -1,3 +1,16 @@
 ## Strategies {#strategies}
 
-_This section needs content_
+Tools may optionally have multiple strategies of operation. `BaseTool`, from which all tools inherit, takes two arguments related to the tools strategies:
+
+| Argument | Type | Description |
+|----------|------|-------------|
+| strategies | Object |An `Object` comprised of `function`s that take `evt` as an argument and perform an operation.
```

Comment:
As this function is being created outside of the tool class, it also has access as a parameter to `tool.configuration` so it can have access to it values.